src/CMakeLists.txt

```diff
@@ -229,3 +177,22 @@ target_include_directories(parthenon PUBLIC
 )
 
 lint_target(parthenon)
+
+# Generate install targets for make if CMAKE_INSTALL_PREFIX has been provided
+if(NOT CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  install(TARGETS parthenon LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
```

Comment:
If you're going to solve install, please also install a `ParthenonConfig.cmake` file to make `find_package(Parthenon)` work in CMake.

See: https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html